components/button/demo/button-toggle.html

```diff
@@ -54,6 +54,26 @@ <h2>Toggle Button (disabled)</h2>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>Toggle Button (with promise delay)</h2>
```

Comment:
Seems like we could remove reference to "promise" here since there's no promise involved. Could even set it up with `d2l-dialog-confirm`, but the `setTimeout` is simple and effective. 😏